src/codegen/mod.rs

```diff
@@ -1202,9 +1202,9 @@ impl CodeGenerator for CompInfo {
         let mut methods = vec![];
         let mut anonymous_field_count = 0;
         for field in struct_fields {
-            debug_assert_eq!(current_bitfield_width.is_some(),
+            extra_assert_eq!(current_bitfield_width.is_some(),
                              current_bitfield_layout.is_some());
-            debug_assert_eq!(current_bitfield_width.is_some(),
+            extra_assert_eq!(current_bitfield_width.is_some(),
```

Comment:
I'd prefer to keep the cheap ones `debug_assert_eq!`s, etc, wdyt?

I believe this could hide real bugs.